lib/utils/messages.ts

```diff
@@ -15,26 +15,20 @@
  * limitations under the License.
  */
 
-const DEBUG_MODE = !!process.env.DEBUG;
-
 export const reachingOut = `\
-If you need help, or just want to say hi, don't hesitate in reaching out
-through our discussion and support forums at https://forums.balena.io
-
-For bug reports or feature requests, have a look at the GitHub issues or
-create a new one at: https://github.com/balena-io/balena-cli/issues/\
+For further help or support, visit:
+https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
 `;
 
 const debugHint = `\
 Additional information may be available with the \`--debug\` flag.
-`;
+\n`;
 
-export const help = `\
-For help, visit our support forums: https://forums.balena.io
-For bug reports or feature requests, see: https://github.com/balena-io/balena-cli/issues/
-`;
+export const help = reachingOut;
```

Comment:
In case of **unexpected** errors only:
_With --debug:_
```
$ ./bin/balena login --debug
...
You don't have sufficient privileges to run this operation.
Try running this command again prefixing it with `sudo`.

If this is not the case, and you're trying to burn an SDCard, check that the write lock is not set.

Error: EACCES: permission denied, open '/Users/paulo/.balena/token'

For further help or support, visit:
https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
```
_Without --debug:_
```
$ ./bin/balena login
...
You don't have sufficient privileges to run this operation.
Try running this command again prefixing it with `sudo`.

If this is not the case, and you're trying to burn an SDCard, check that the write lock is not set.

Additional information may be available with the `--debug` flag.

For further help or support, visit:
https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
```